presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/mysql/FunctionNamespaceDao.java

```diff
@@ -50,23 +50,23 @@
     void createFunctionNamespacesTableIfNotExists();
 
     @SqlUpdate("CREATE TABLE IF NOT EXISTS <sql_functions_table> (\n" +
-            "  id bigint(20) NOT NULL AUTO_INCREMENT,\n" +
-            "  function_id varchar(700) NOT NULL,\n" +
+            "  function_id_hash varchar(128) NOT NULL,\n" +
```

Comment:
Where did `id` go?

Reply:
With `id` column removed, this table will have no primary key, and mysql will automatically creates an inaccessible column to use as the primary key column, so why do we bother to remove it. 😂 Added it back.